### ISSUE TITLE
Should the Grinding Stone be a container, automatically grinding Flour and Shurger when spinning?

### ISSUE THREAD

**Issue by Kaiochao** [Kaiochao/HazordhuII#46]
Maybe even sharpen things automatically... but that wouldn't make as much sense.

**Comment by Unwanted4Murder** [comment]
++

I don't think it should sharpen automatically though.  What if there were two different types of grinders instead?

**Comment by Kaiochao** [comment]
True. Instead of replacing the existing Grinding Stone, it could be a separate, more expensive upgraded version. Same with the container-style Forge.
